### PR TITLE
Add new zh-tw localizations & move non-legacy localization keys

### PR DESCRIPTION
**Pull request by Still34**
# Summary

This PR updates the localization table for zh-tw, as well as fixing inconsistencies in the existing tables. Most notably, moving stats exclusive keys under the stats category, as opposed to leaving them in the root. Other similar entries are kept in the root as they are considered strings that are shared across the entire UI (e.g., "Scenes" may be used on the stats screen, on the navbar, etc.) Additionally, key naming inconsistencies are also corrected, where some key names used the kebab-case naming conventions instead of the snake-case one used by the rest of the table.

### CODE REVIEW

**Comment by WithoutPants** [comment]
There's been a few localisation updates in the meantime, so this will need to be updated.

**Comment by Still34** [comment]
Updated to the latest develop branch and made fixes from the conflicts.

File: ui/v2.5/src/locales/en-GB.json [hunk 1]
```diff
@@ -237,7 +237,9 @@
       "sqlite_location": "File location for the SQLite database (requires restart)",
       "video_ext_desc": "Comma-delimited list of file extensions that will be identified as videos.",
       "video_ext_head": "Video Extensions",
-      "video_head": "Video"
+      "video_head": "Video",
+      "include_audio_head": "Include audio",
+      "include_audio_desc": "Includes audio stream when generating previews."
```
Comment (WithoutPants) [inline ui/v2.5/src/locales/en-GB.json:242]: What are these? I don't see any reference in the tsx files.

Comment (Still34) [reply ui/v2.5/src/locales/en-GB.json:242]: Huh, it's supposed to be under the generated content settings. I thought I replaced the string in there. I'll fix it when I'm home, or you could make a commit. Either is fine by me.

**Review by WithoutPants — APPROVED** [review]

**Pull request merged**